site/gatsby-site/src/pages/cite/edit.js

```diff
@@ -179,13 +181,15 @@ function EditCitePage(props) {
 
         const embedding = incidentEmbedding(exSiblings);
 
-        await client.mutate({
-          mutation: UPDATE_INCIDENT,
-          variables: {
-            query: { incident_id: parentIncident.incident.incident_id },
-            set: { embedding },
-          },
-        });
+        if (embedding) {
```

Comment:
Looks like this PR does something else besides the title, whats the `embedding` check for?

Reply:
I think when I was testing that the toasts appeared correctly, I noticed that the form couldn't be submitted when the incident being edited didn't have an embedding. Since the NLP api sometimes fails to return a result, we can't count on there being one.